src/data_classifier.py

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env python
```

Comment:
Just noticed, this line is only needed if you want to run it from command line, should be safe to remove it.

Reply:
I think you removed it from `reduce_data.py`, though we do launch that one with Python from the command line. I think it's just src/data_classifier.py that doesn't need it.

Reply:
Oh yes, sorry! I haven't pushed that change yet, but yes, it's back :)